### PR TITLE
design auth screens

### PR DESCRIPTION
**Pull request by salehanoor644**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -1,27 +1,29 @@
 import React from 'react';
-import { TouchableOpacity, View, Text } from 'react-native';
+import { TouchableOpacity, View } from 'react-native';
 //@ts-ignore
 import AntDesign from 'react-native-vector-icons/AntDesign';
-import mainStyle from '@src/constants/MainStyles';
+import { Text } from '../../../blueprints/Text/Text';
```
Comment (Mubashar-Stack) [inline src/components/Header/Header.tsx]: Don't use this dot dot while accessing the blueprints instead you can use this `@app/blueprints`

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
-   <TouchableOpacity style={{ width: '16%', justifyContent: 'center', alignItems: 'center' }} onPress={onPress} >
-    <AntDesign size={18} name={"left"} color={'#000000'} />
-   </TouchableOpacity>
-   <Text style={{ ...mainStyle.headerText, color: '#000000', }}>{title}</Text>
-  </View>
-
- );
+  return (
+    <View style={{ flexDirection: 'row', alignSelf: 'center', marginVertical: 14, alignItems: 'center', width: '90%' }}>
+      {title == 'Sign Up' || title == 'Log In' ? (
```
Comment (Mubashar-Stack) [inline src/components/Header/Header.tsx]: Refactored Header component for improved condition handling and TypeScript support. Example: replaced inline condition with `isAuthScreen` variable.
`const isAuthScreen = ['Sign Up', 'Log In'].includes(title);`

File: src/constants/MainStyles.tsx [hunk 3]
```diff
@@ -58,25 +30,25 @@ const mainStyle = StyleSheet.create({
...
   borderRadius: 16,
-  backgroundColor: '#ccc',
+  backgroundColor: color.Tertiary,
   paddingHorizontal: 10,
   height: 52,
+  alignItems: 'center', justifyContent: 'space-between'
  },
  inputText: {
-  fontFamily: Platform.OS == 'ios' ? 'GEDinarOne-Medium' : 'Montserrat-Medium',
+  fontFamily: 'Montserrat-Medium',
```
Comment (Mubashar-Stack) [inline src/constants/MainStyles.tsx]: Use this font (Roboto)  as per figma

File: src/navigation/AppNavigation.tsx [hunk 4]
```diff
@@ -44,10 +37,7 @@ export const AppNavigation = () => {
       ) : (
         <Stack.Navigator screenOptions={screenOptions}>
           <Stack.Screen name={Screen.NEWS_LIST} component={NewsListScreen} />
-          <Stack.Screen
-            name={Screen.NEWS_DETAIL}
-            component={NewsDetailScreen}
-          />
+          <Stack.Screen name={Screen.NEWS_DETAIL} component={NewsDetailScreen} />
```
Comment (Mubashar-Stack) [inline src/navigation/AppNavigation.tsx]: Remove extra screens from code and navigation stack as well

File: src/screens/Login/Login.style.ts [hunk 5]
```diff
@@ -1,31 +1,99 @@
...
+//   StyleSheet.create({
+//     container: {
+//       alignItems: 'center',
+//       flex: 1,
+//       justifyContent: 'center',
+//     },
+//     content: {
+//       backgroundColor: backgroundColor,
+//       borderRadius: scaledSize(30),
+//       flex: 2,
```
Comment (Mubashar-Stack) [inline src/screens/Login/Login.style.ts]: Please avoid commenting out code like this. If it's not needed, remove it entirely.

File: src/screens/Login/LoginScreen.tsx [hunk 6]
```diff
@@ -1,108 +1,112 @@
...
 //@ts-ignore
 import Feather from 'react-native-vector-icons/Feather';
-import { Screen } from '../../navigation/appNavigation.type';
+//@ts-ignore
+import Ionicons from 'react-native-vector-icons/Ionicons';
+import { useAppContext, useColor } from '@src/context';
+import mainStyle from '@src/constants/MainStyles';
 import Header from '@src/components/Header/Header';
+import { Formik } from 'formik';
+import { Screen } from '../../navigation/appNavigation.type'
```
Comment (Mubashar-Stack) [inline src/screens/Login/LoginScreen.tsx:12]: Here is the alias list. Please use these, with the default being src. Start with `@src` and follow the structure onward
```
alias: {
  '@app/blueprints': './blueprints',
  '@src/assets': './src/assets',
  '@src/components': './src/components',
  '@src/constants': './src/constants',
  '@src/context': './src/context',
  '@src/hooks': './src/hooks',
  '@src/i18n': './src/i18n',
  '@src/screens': './src/screens',
  '@src/services': './src/services',
  '@src/store': './src/store',
  '@src/utils': './src/utils',
}
```

**Review by Mubashar-Stack — CHANGES_REQUESTED** [review]
I've reviewed your code, so please check the comments and address the issues. Also, ensure the screens follow the Figma flow — for example, the initial screen is missing. I'll run the code locally this evening and let you know if any further improvements are needed

File: src/components/Header/Header.tsx (hunk 1 above)
Comment (mirajkhandev) [reply src/components/Header/Header.tsx:5]: okay

File: src/components/Header/Header.tsx (hunk 2 above)
Comment (mirajkhandev) [reply src/components/Header/Header.tsx:16]: Done

**Review by Mubashar-Stack — APPROVED** [review]

**Pull request merged**